{
    "affected": [
        {
            "package": {
                "ecosystem": "Photon OS:4.0",
                "name": "linux",
                "purl": "pkg:rpm/vmware/linux?distro=photon-4"
            }
        },
        {
            "package": {
                "ecosystem": "Photon OS:4.0",
                "name": "linux-aws",
                "purl": "pkg:rpm/vmware/linux-aws?distro=photon-4"
            }
        },
        {
            "package": {
                "ecosystem": "Photon OS:4.0",
                "name": "linux-rt",
                "purl": "pkg:rpm/vmware/linux-rt?distro=photon-4"
            }
        },
        {
            "package": {
                "ecosystem": "Photon OS:4.0",
                "name": "linux-secure",
                "purl": "pkg:rpm/vmware/linux-secure?distro=photon-4"
            }
        }
    ],
    "id": "PHSA-2024-4.0-0588",
    "modified": "2024-04-05T05:24:50Z",
    "published": "2024-04-02T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-4.0-588"
        }
    ],
    "related": [
        "CVE-2023-52585"
    ]
}